src/lib/openapi/spec/bootstrap-ui-schema.test.ts

```diff
@@ -0,0 +1,152 @@
+                stickiness: false,
+                sortOrder: 0,
+                legalValues: [],
+                createdAt: '2022-04-08T10:59:24.374Z',
+            },
+            {
+                name: 'userId',
+                description: 'Allows you to constrain on userId',
+                stickiness: false,
+                sortOrder: 1,
+                legalValues: [],
+                createdAt: '2022-04-08T10:59:24.374Z',
+            },
+        ],
+        featureTypes: [
+            {
+                id: 'release',
+                name: 'Release',
+                description:
+                    'Release feature toggles are used to release new features.',
+                lifetimeDays: 40,
+            },
+            {
+                id: 'experiment',
+                name: 'Experiment',
+                description:
+                    'Experiment feature toggles are used to test and verify multiple different versions of a feature.',
+                lifetimeDays: 40,
+            },
+            {
+                id: 'operational',
+                name: 'Operational',
+                description:
+                    'Operational feature toggles are used to control aspects of a rollout.',
+                lifetimeDays: 7,
+            },
+        ],
+        tagTypes: [
+            {
+                name: 'simple',
+                description: 'Used to simplify filtering of features',
+                icon: '#',
+            },
+            { name: 'hashtag', description: '', icon: null },
+        ],
+        strategies: [
+            {
+                displayName: 'Standard',
+                name: 'default',
+                editable: false,
+                description:
+                    'The standard strategy is strictly on / off for your entire userbase.',
+                parameters: [],
+                deprecated: false,
+            },
+            {
+                displayName: null,
+                name: 'gradualRolloutRandom',
+                editable: true,
+                description:
+                    'Randomly activate the feature toggle. No stickiness.',
+                parameters: [
+                    {
+                        name: 'percentage',
+                        type: 'percentage',
+                        description: '',
+                        required: false,
+                    },
+                ],
+                deprecated: true,
+            },
+        ],
+        projects: [
+            {
+                name: 'Default',
+                id: 'default',
+                description: 'Default project',
+                health: 74,
+                featureCount: 10,
+                memberCount: 3,
+                updatedAt: '2022-06-28T17:33:53.963Z',
+            },
+        ],
+    };
+
+    expect(
+        validateSchema('#/components/schemas/bootstrapUiSchema', data),
+    ).toBeUndefined();
```

Comment:
Could add another test to ensure that the validation at least returns some error:

```
expect(
    validateSchema('#/components/schemas/bootstrapUiSchema', {}),
).not.toBeUndefined();
```